### PR TITLE
Add tips for CI signal team on working with other SIGs

### PR DESCRIPTION
**Pull request by mariantalla**
This PR adds more detail on what sort of work with other SIGs the CI signal team can support with.

Related: https://github.com/kubernetes/community/pull/3143 ; which adds information for contributors (not necessarily on the CI signal team) to triage and escalate test failures.

cc @jberkus 

@spiffxp @BenTheElder :  A lot of the content in this PR comes from the testing.md file mentioned in kubernetes/sig-release#428 , and it's points I thought were mostly useful to CI signal but not as much to other contirbutors. Thoughts? Anything to add/remove/reword?

### CODE REVIEW

File: release-team/role-handbooks/ci-signal/README.md [hunk 1]
```diff
@@ -154,6 +155,23 @@ The CI Signal lead should
...
+
+1. Identifying tests from sig-<name> that should be/are part of sig-release's blocking and informing dashboards. Those tests could be submitted as part of a new feature/enhancement that sig-<name> is developing, or could be existing tests in blocking/informing dashboards.
+Questions to ask sig-<name>:
+  - Which e2e test jobs are release blocking for your SIG?
+  - What is the process for making sure the SIGs test grid remains healthy and resolving test failures?
+  - Would moving the e2e tests for the SIG into their own test jobs make maintaining those tests easier? If yes, consider placing them in a dashboard owned by sig-<name>.
+  - Is there a playbook for how to resolve test failures and how to identify whether or not another SIG owns the resolution of the issue? If not, could you (sig-<name>) develop one?
+  - What is the escalation point (email + slack) that will be responsible for keeping this test healthy?
+
+1.Escalating test failures/flakes to sig-<name>. Expectations:
```
Comment (hoegaarden) [inline release-team/role-handbooks/ci-signal/README.md]:
```suggestion
1. Escalating test failures/flakes to sig-<name>. Expectations:
```

File: release-team/role-handbooks/ci-signal/README.md [hunk 2]
```diff
@@ -154,6 +155,23 @@ The CI Signal lead should
...
 
+## Working with SIGs outside sig-release
+2 scenarios that you will be involved in:
+
+1. Identifying tests from sig-<name> that should be/are part of sig-release's blocking and informing dashboards. Those tests could be submitted as part of a new feature/enhancement that sig-<name> is developing, or could be existing tests in blocking/informing dashboards.
+Questions to ask sig-<name>:
+  - Which e2e test jobs are release blocking for your SIG?
+  - What is the process for making sure the SIGs test grid remains healthy and resolving test failures?
```
Comment (hoegaarden) [inline release-team/role-handbooks/ci-signal/README.md]:
```suggestion
  - What is the process for making sure the SIG's test grid remains healthy and resolving test failures?
```

File: release-team/role-handbooks/ci-signal/README.md [hunk 3]
```diff
@@ -154,6 +155,23 @@ The CI Signal lead should
...
+  - Would moving the e2e tests for the SIG into their own test jobs make maintaining those tests easier? If yes, consider placing them in a dashboard owned by sig-<name>.
+  - Is there a playbook for how to resolve test failures and how to identify whether or not another SIG owns the resolution of the issue? If not, could you (sig-<name>) develop one?
+  - What is the escalation point (email + slack) that will be responsible for keeping this test healthy?
+
+1. Escalating test failures/flakes to sig-<name>. Expectations:
+  - Each test must have an escalation point (email + slack).  The escalation point is responsible for keeping the test healthy.
+  - Fixes for test failures caused by areas of ownership outside the responsibility of the escalation point should be coordinated with other teams by the test escalation point.
+  - Escalation points are expected to be responsive within 24 hours, and prioritize test failure issues over other issues.
```
Comment (hoegaarden) [inline release-team/role-handbooks/ci-signal/README.md]:
```suggestion
  - Escalation points are expected to be responsive within 24 hours, and to prioritize test failure issues over other issues.
```

**Comment by justaugustus** [comment]
@mariantalla -- bump on this one

**Comment by mariantalla** [comment]
Whoops thanks for the poke - now open for review 🙏🏻

File: release-team/role-handbooks/ci-signal/README.md [hunk 4]
```diff
@@ -154,6 +155,23 @@ The CI Signal lead should
...
   - Yet another tip to assess seriousness of the failure is to see if both gce and gke counterparts of a upgrade/downgrade job are failing eg: [gce-1.10-master-upgrade-master](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gce-1.10-master-upgrade-master) and [gke-gci-1.10-gci-master-upgrade-master](https://k8s-testgrid.appspot.com/sig-release-master-upgrade#gke-gci-1.10-gci-master-upgrade-master). If both are failing then its indicative of a systemic failure in upgrade functionality. If only gke jobs are failing then it might be a GKE only issue and does not block k8s release. In this case escalate to GKE Cluster Lifecycle team  (@roberthbailey, @krousey)
 
+## Working with SIGs outside sig-release
+2 scenarios that you will be involved in:
+
+1. Identifying tests from sig-<name> that should be/are part of sig-release's blocking and informing dashboards. Those tests could be submitted as part of a new feature/enhancement that sig-<name> is developing, or could be existing tests in blocking/informing dashboards.
+Questions to ask sig-<name>:
+  - Which e2e test jobs are release blocking for your SIG?
```
Comment (spiffxp) [inline release-team/role-handbooks/ci-signal/README.md]: Most of them won't know, but I get the line of questioning.  In the case of catchall jobs (eg [release-master-blocking/gce-cos-master-default](https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-default)), I suspect we can at best ask that they are aware whether their sig has tests that run as part of those jobs.

File: release-team/role-handbooks/ci-signal/README.md [hunk 5]
```diff
@@ -154,6 +155,23 @@ The CI Signal lead should
...
+  - What is the process for making sure the SIG's test grid remains healthy and resolving test failures?
+  - Would moving the e2e tests for the SIG into their own test jobs make maintaining those tests easier? If yes, consider placing them in a dashboard owned by sig-<name>.
+  - Is there a playbook for how to resolve test failures and how to identify whether or not another SIG owns the resolution of the issue? If not, could you (sig-<name>) develop one?
+  - What is the escalation point (email + slack) that will be responsible for keeping this test healthy?
+
+1. Escalating test failures/flakes to sig-<name>. Expectations:
+  - Each test must have an escalation point (email + slack).  The escalation point is responsible for keeping the test healthy.
```
Comment (spiffxp) [inline release-team/role-handbooks/ci-signal/README.md]: Where are these escalation points defined/enumerated

**Comment by BenTheElder** [comment]
^ this is due to https://github.com/kubernetes/test-infra/issues/11204

**Comment by mariantalla** [comment]
@spiffxp 

> If you're saying the content in https://github.com/kubernetes/sig-release/blob/master/ephemera/testing.md is only applicable to CI signal staff I respectfully disagree. I think any kubernetes developer should be capable of raising, triaging and escalating test failures. A guide on how to do so in kubernetes/community would help.

Yep 100% with you on that! The other "half" of this PR is https://github.com/kubernetes/community/pull/3143 and is targetted at non-CI-signal-affiliated contributors (edited this PR's decription to link to [community/3143](https://github.com/kubernetes/community/pull/3143) too). The content I added here was mostly related to the `sig-release` dashboard, which I think is not typically looked at by folks outside SIG-release (not sure if this is a valid assumption though 🤔). Thoughts?

File: release-team/role-handbooks/ci-signal/README.md (hunk 5 above)
Comment (mariantalla) [reply release-team/role-handbooks/ci-signal/README.md]: Good point 🤔 I'm thinking:
- Once https://github.com/kubernetes/community/pull/3143 merges they will be in [kubernetes/community/devel/testing.md](https://github.com/kubernetes/community/blob/2720cc0ae1b1bd4a16dbf89c90a4d4393fa299a9/contributors/devel/testing.md) > `Escalating failures to a SIG`. I'll add a todo to link from the CI signal handbook to the contributor guide once that's done.
- Once work for https://github.com/kubernetes/sig-release/issues/441 is complete, `sig-release`'s tests will have more explicit owners+contact details assigned. I'll add another todo to link to `k/test-infra/testgrid/config.yaml` post https://github.com/kubernetes/sig-release/issues/441 closure.

Thoughts?

**Comment by mariantalla** [comment]
@spiffxp assigning you but also holding for a little while as I want to hear your feedback on https://github.com/kubernetes/sig-release/pull/467#issuecomment-462271184 before merging; reason being that I'd like to keep this PR as lean as possible & relevant to CI-signal-ers, and shift anything more broadly useful to the community contribution guide.

/hold
/assign @spiffxp

File: release-team/role-handbooks/ci-signal/README.md (hunk 5 above)
Comment (spiffxp) [reply release-team/role-handbooks/ci-signal/README.md]: It's a start, it still feels like it's spread around a few places.  How am I going to find an e-mail for a SIG if I'm trying to ping them about a specific test case?

Maybe it'd be better to say each test must be owned by a SIG, and the SIG is responsible for providing escalation points.  If a SIG doesn't provide specific escalation points for resolving test failures, the default would be general contact info for the SIG: its chairs, their slack channel, their mailing list.  Where should the SIG detail escalation points? Maybe sigs.yaml or its readme?

In terms of non-defaults, the only things we'll have soon are:
- For release-blocking jobs, the owning SIG and mailing list is determined by the alert options in testgrid's config.

**Comment by spiffxp** [comment]
/lgtm
I'll leave it to you to drop the /hold or iterate, I'm fine with this merging as is and refining.  Thanks for the pointer to k/community.

**Comment by BenTheElder** [comment]
Thanks!

**Pull request merged**